### PR TITLE
Copy Collection to repositories fix

### PR DESCRIPTION
**Pull request by MilanPospisil**
Fixing broken copy collection to repositories.

This PR also adds refresh table after modal is closed.

### CODE REVIEW

File: frontend/hub/collections/hooks/useCopyToRepository.tsx [hunk 1]
```diff
@@ -233,7 +236,7 @@ export async function copyToRepositoryAction(
   }
 
   const pipeline = collection.repository?.pulp_labels?.pipeline;
-  if (!(pipeline === 'staging' || pipeline === 'rejected')) {
+  if (operation === 'approve' && !(pipeline === 'staging' || pipeline === 'rejected')) {
```
Comment (MilanPospisil) [inline frontend/hub/collections/hooks/useCopyToRepository.tsx:239]: This caused the problems, the approvals have this condition to allow only approvals from staging or rejected, but it was failing in copy (which is allowed in UI only from approved repos).

**Comment by akus062381** [comment]
Thanks for adding this!
I am testing your PR now, and see the following behavior:

1. Upload a collection
2. About 10 repositories already exist (if not add about 10 repositories)
3. Copy the collection into all 10 repositories
4. See the new list of collections in all the repositories
5. Try to delete all those collections, including the one you originally uploaded
6. See the following in the UI:
image

**Review by akus062381 — CHANGES_REQUESTED** [review]
Can we find a way to handle bulk deletion of multiple copied collections?

**Comment by MilanPospisil** [comment]
> Can we find a way to handle bulk deletion of multiple copied collections?

Lets solve it in separate PR, this is unrelated problem and I know why it happends.

**Review by akus062381 — APPROVED** [review]

**Pull request merged**